doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,599 @@
+   device. Select **UEFI: SanDisk**.
+#. Install the Ubuntu OS.
+#. Select **Something else** to create the partition.
+
+   .. figure:: images/native-ubuntu-on-SATA-1.png
+
+#. Configure the ``/dev/sda`` partition. Refer to the diagram below:
+
+   .. figure:: images/native-ubuntu-on-SATA-2.png
+
+   a. Select the ``/dev/sda`` partition, not ``/dev/nveme0p1``.
+   b. Select ``/dev/sda`` **ATA KINGSTON RBUSNS4** as the device for the
+      boot loader installation.
+
+#. Continue with the Ubuntu Service VM installation in /dev/sda.
+
+Install the Ubuntu Service VM on the NVME disk
+----------------------------------------------
+
+.. note:: Before you install the Ubuntu Service VM on the NVME disk, either
+   remove the SATA disk or disable it in the BIOS. Disable it by going to:
+   **Chipset** → **PCH-IO Configuration** -> **SATA and RST Configuration** -> **SATA Controller [Disabled]**
+
+#. Insert the Ubuntu USB boot disk into the WHL Maxtang machine.
+#. Power on the machine, then press F11 to select the USB disk as the boot
+   device. Select **UEFI: SanDisk**.
+#. Install the Ubuntu OS.
+#. Select **Something else** to create the partition.
+
+   .. figure:: images/native-ubuntu-on-NVME-1.png
+
+#. Configure the ``/dev/nveme0n1`` partition. Refer to the diagram below:
+
+   .. figure:: images/native-ubuntu-on-NVME-2.png
+
+   a. Select the ``/dev/nveme0n1`` partition, not ``/dev/sda``.
+   b. Select ``/dev/nveme0n1`` **FORESEE 256GB SSD** as the device for the
+      boot loader installation.
+
+#. Complete the Ubuntu installation and reboot the system.
+
+   .. note:: Set **acrn** as the username for the Ubuntu Service VM.
+
+
+Build and Install ACRN on Ubuntu
+********************************
+
+Pre-Steps
+=========
+
+#. Set the network configuration, proxy, etc.
+#. Update Ubuntu:
+
+   .. code-block:: none
+
+      $ sudo -E apt update
+
+#. Create a work folder:
+
+   .. code-block:: none
+
+      $ mkdir /home/acrn/work
+
+Build the ACRN Hypervisor on Ubuntu
+===================================
+
+#. Install the necessary libraries:
+
+   .. code-block:: none
+
+      $ sudo -E apt install gcc \
+        git \
+        make \
+        gnu-efi \
+        libssl-dev \
+        libpciaccess-dev \
+        uuid-dev \
+        libsystemd-dev \
+        libevent-dev \
+        libxml2-dev \
+        libusb-1.0-0-dev \
+        python3 \
+        python3-pip \
+        libblkid-dev \
+        e2fslibs-dev \
+        pkg-config \
+        zlib1g-dev \
+        libnuma-dev \
+
+        liblz4-tool
+
+      $ pip3 install kconfiglib
+
+#. Get the ACRN source code:
+
+   .. code-block:: none
+
+      $ cd /home/intel/work
+      $ git clone https://github.com/projectacrn/acrn-hypervisor
+      $ cd acrn-hypvervisor
+
+#. Switch to the v2.0 version:
+
+   .. code-block:: none
+
+      $ git checkout -b v2.0 remotes/origin/release_2.0
+
+4. Apply CAT and other patches if necessary [optional].
```

Comment:
@gvancuts to which lines are you referring? @fuzhongl please verify.

Reply:
Since acrn-hypvervisor source code is keep moving; 
git checkout -b v2.0 remotes/origin/release_2.0 is preferred.
 You can remove `4. Apply CAT and other patches if necessary [optional].
`
 Thanks!

Reply:
Removed!